setup.cfg

```diff
@@ -16,17 +17,18 @@ classifiers =
     Programming Language :: Python :: 3.6
     Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
     Programming Language :: Python :: 3
     Programming Language :: Python :: Implementation :: CPython
     Programming Language :: Python :: Implementation :: PyPy
 platforms = any
 license = MIT
 
 [options]
-scripts = git-filter-repo
 py_modules = git_filter_repo
 python_requires = >= 3.5
-setup_requires = setuptools_scm
```

Comment:
`setup_requires` is [deprecated in favor of](https://setuptools.pypa.io/en/latest/references/keywords.html) `pyproject.toml`